packages/api/src/middleware/expoJudgeAccessMiddleware.ts

```diff
@@ -0,0 +1,43 @@
+import { ExpoJudgingSession, ExpoJudgingSessionContext } from '@hangar/database';
+import { NextFunction, Request, Response } from 'express';
+import { logger } from '../utils/logger';
+
+export const expoJudgeAccessMiddleware = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const {
+    judge,
+    entityManager: em,
+    params: { id: ejsId },
+  } = req;
+
+  try {
+    const ejsCount = await em.count(ExpoJudgingSession, { id: ejsId as string });
+    if (ejsCount < 0) {
+      res.sendStatus(404);
+      return;
+    }
+
+    const contextCount = await em.count(ExpoJudgingSessionContext, {
+      expoJudgingSession: {
+        id: ejsId,
+      },
+      judge: {
+        id: judge.id,
+      },
+    });
+
+    if (!contextCount) {
+      res.sendStatus(403);
+      return;
+    }
+
+    res.sendStatus(204);
```

Comment:
This will result in an error for any handler and brick the ability to send data back from within a handler